src/pages/about-us/index.tsx

```diff
@@ -12,106 +12,109 @@ const AboutUs = () => {
   return !user ? (
     <Loader />
   ) : (
-    <div className={styles.aboutUsPage}>
-      <div className={styles.pageTitle}>
-        <h2>About the Space Force</h2>
-      </div>
+    <>
```

Comment:
nit-pick: Looks like this wrapper object is not really needed anymore. But It doesn't really hurt anything having it either.